tests/client_tests/python/asyncpg/test_asyncpg.py

```diff
@@ -80,6 +80,18 @@ async def bitstring_can_be_inserted_and_selected_using_binary_encoding(test, con
     test.assertEqual(result[0][0], xs)
 
 
+async def float_vector_can_be_inserted_and_selected(test, conn):
+    xs = [1.1, 2.2, 3.3, 4.4]
+    await conn.execute('drop table if exists tbl_fv')
+    await conn.execute('create table tbl_fv (id int, xs float(4))')
+    await conn.execute('insert into tbl_fv (xs) values (1, ?)', xs)
+    await conn.execute('insert into tbl_fv (xs) values (2, ?)', None)
+    await conn.execute('refresh table tbl_fv')
+    result = await conn.fetch('select xs from tbl_fv order by id')
+    test.assertEqual(result[0][0], xs)
+    test.assertNone(result[0][0])
```

Comment:
Shouldn't it be `test.assertNone(result[1][0])`?

Reply:
yes, thank you! I have to check